views/pages/details.html

```diff
@@ -106,6 +106,10 @@
     </tbody>
   </table>
   <a id="newSearchLink" href="/" class="button">New search</a>
+  <a id="editSearchLink" href="/?edit&{{querystring}}" class="blue button">Edit search</a>
```

Comment:
Technically we should URL escape the & to &amp; But that is a mistake I made in another PR can be fixed another time if need be.